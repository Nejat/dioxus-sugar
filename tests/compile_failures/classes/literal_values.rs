use dioxus_sugar::classes;

#[classes(42)]
struct PropsInteger {}

#[classes("42")]
struct PropsString {}

fn main() {}