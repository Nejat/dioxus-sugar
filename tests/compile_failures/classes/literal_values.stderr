error: literals are not supported
 --> tests/compile_failures/classes/literal_values.rs:3:11
  |
3 | #[classes(42)]
  |           ^^

error: literals are not supported
 --> tests/compile_failures/classes/literal_values.rs:6:11
  |
6 | #[classes("42")]
  |           ^^^^
